awx/main/access.py

```diff
@@ -2781,10 +2781,15 @@ def can_delete(self, obj):
 
 class WorkflowApprovalAccess(BaseAccess):
     '''
```

Comment:
We might want to add something like:

A user can see approvals if have read access to the associated WorkflowJobTemplate.